rust/private/rustfmt.bzl

```diff
@@ -134,6 +149,48 @@ generated source files are also ignored by this aspect.
             default = Label("//util/process_wrapper"),
         ),
     },
+    incompatible_use_toolchain_transition = True,
+    required_providers = [
+        [rust_common.crate_info],
+        [rust_common.test_crate_info],
+    ],
+    fragments = ["cpp"],
+    host_fragments = ["cpp"],
+    toolchains = [
+        str(Label("//rust/rustfmt:toolchain_type")),
+    ],
+)
+
+def _rustfmt_test_manifest_aspect_impl(target, ctx):
+    crate_info = _get_rustfmt_ready_crate_info(target)
+
+    if not crate_info:
+        return []
+
+    # Parse the edition to use for formatting from the target
+    edition = crate_info.edition
+
+    srcs = _find_rustfmtable_srcs(crate_info, ctx)
+    manifest = _generate_manifest(edition, srcs, ctx)
+
+    return [
+        OutputGroupInfo(
+            rustfmt_manifest = depset([manifest]),
+        ),
+    ]
+
+# This aspect contains functionality split out of `rustfmt_aspect` which broke when
```

Comment:
This feels like a bug upstream. @scentini have you ever encountered this?

Reply:
Nope, but I could check and file a bug against Bazel if the claim still holds.

Reply:
https://github.com/bazelbuild/bazel/issues/17214